.eslintrc.json

```diff
@@ -16,14 +16,17 @@
     "prettier",
     "mongodb"
   ],
+  "ignorePatterns": [
+    "src/seed/"
+  ],
   "rules": {
     "import/prefer-default-export": "off",
     "import/no-default-export": "error",
     "mongodb/no-replace": 0,
     "no-unused-vars": [
       "error",
       {
-        "argsIgnorePattern": "next"
+        "args": "none"
```

Comment:
could you shortly explain these please?

Reply:
It is to change how the eslint rule works with arguments, did you try to find it? [They](https://eslint.org/docs/rules/no-unused-vars#args) explain it better than me